Hardcore.lua

```diff
@@ -207,6 +207,14 @@ local ALERT_STYLES = {
 		delay = 10,
 		alertSound = 8192
 	},
+	videre_warning = {
+		frame = Hardcore_Alert_Frame,
+		text = Hardcore_Alert_Text,
+		icon = Hardcore_Alert_Icon,
+		file = "alert-hc-red.blp",
+		delay = 10,
```

Comment:
If we're making a custom config for this, can we make the delay much less? Like 1-2 seconds?

Reply:
Delay is how long it's displayed right? It's a fairly long message, so figured it's good to have it stay up for a little while.